src/js/redux/reducers/index.js

```diff
@@ -50,7 +51,8 @@ const appReducer = combineReducers({
     tas: tasReducer,
     psc: pscReducer,
     searchMapLegendToggle: mapLegendToggleReducer,
-    agencyV2: agencyV2Reducer
+    agencyV2: agencyV2Reducer,
+    covid19: covid19Reducer
```

Comment:
In all seriousness this time... do we want to make this more generic?

Reply:
dont think so. I think @ebdabbs suggested `covid19` during a slack call. But we can name it anything.